x-pack/plugins/security_solution/public/timelines/components/timeline/body/events/event_column_view.tsx

```diff
@@ -151,13 +151,17 @@ export const EventColumnView = React.memo<Props>(
               />,
             ]
           : []),
-        <AlertContextMenu
-          key="alert-context-menu"
-          ecsRowData={ecsData}
-          timelineId={timelineId}
-          disabled={eventType !== 'signal'}
-          refetch={refetch}
-        />,
+        ...(eventType !== 'raw'
```

Comment:
It's preferable to rely on the existing `disabled={eventType !== 'signal'}` check, which disables the context menu for regular events, because conditionally rendering it leaves a "blank spot" for non-alerts, per the screenshot below:

<img width="165" alt="empty-space" src="https://user-images.githubusercontent.com/4459398/94865531-0033c700-03fb-11eb-9743-5f926b61c680.png">

Reply:
This one can also be addressed in a follow-up PR with the ellipsis `...` paging control change